README.md

```diff
@@ -28,7 +28,7 @@ Create a new WebSocket connection using the following.
 - Password is optional.  
 
 ```js
-const obs = new OBSWebSocket('address', 'password');
+const obs = new OBSWebSocket({ address: 'localhost:4444', password: '$up3rSecretP@ssw0rd' });
```

Comment:
Sure, we could have a chat with them if nothing else. Feel free to have them send an email to info@supportclass.net.